src/main/java/kg/jarkyn/cobspecserver/Server.java

```diff
@@ -1,5 +1,9 @@
 package kg.jarkyn.cobspecserver;
 
+import kg.jarkyn.cobspecserver.middleware.ResponseController;
+import kg.jarkyn.cobspecserver.sockets.ClientSocket;
+import kg.jarkyn.cobspecserver.sockets.ListenerSocket;
+
```

Comment:
this is an interesting question, this happened when I organised the code into packages. Before that they were all part of the ```kg.jarkyn.cobspecserver```
Server was the first thing I implemented as I approached the problem top-down, hence the collaborators were more or less decided since the beginning.

Here's the commit where Server was first implemented
https://github.com/indykid/cobSpecServer/commit/b46e447501de1067446595adec1d58b01afaea76